src/components/Note/NoteAuthorList/NoteAuthorList.scss

```
@@ -0,0 +1,123 @@
+@import "src/constants/style.scss";
+
+$overlap-margin: -26px;
+$name-gap: 4px;
+
+// --self classes: indicates yourself as the stack author,
+// which is highlighted by a blue background.
+
+.note-author-list {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+
+  padding: 2px 14px 2px 2px;
+  border-radius: $rounded--full;
+}
+
+.note-author-list--self {
+  background: var(--accent-color--desaturated-light);
+}
+
+.note__author {
+  display: inline-flex;
+  align-items: center;
+  // reset css from figure-element:
+  border: 0;
+  font: inherit;
+  vertical-align: baseline;
+  flex-grow: 0;
+  height: 28px;
+  background: $color-white;
+  border-radius: $rounded--default;
+
+  padding: 0 $padding--default 0 0;
+  margin: 0 0 0 $overlap-margin;
+}
+
+// reset negative margin on first avatar
+.note__author:first-of-type {
+  margin: 0;
+}
+
+// make a gap after last avatar unless .note-author-rest bubble exists
+.note__author:last-of-type:not(.note__author--with-rest) {
+  padding-right: 0;
+  margin-right: $name-gap;
+}
+
+// the rest bubble should be about the same size as the avatars
+.note-author-rest {
+  display: inline-flex;
+  width: $icon--medium;
+  height: $icon--medium;
+
+  font-size: $text-size--small;
+
```

Please use our new font size constants here 
```suggestion
  font-size: $text--xs;
```